src/helpers/__snapshots__/utils.test.ts.snap

```diff
@@ -230,6 +230,12 @@ Array [
         "link": "http://php.net/manual/en/book.curl.php",
         "title": "cURL",
       },
+      Object {
+        "description": "PHP with guzzle v7",
```

Comment:
just to say it out loud, I don't have a ton of interest in keeping old versions of things around in this library as targets.

so like.. if in the future guzzle releases v8 and it's incompatible with what we have here, I'd rather we just update the guzzle target to v8 (using the same `guzzle` key) and major (or minor?) release `httpsnippet` with only that updated version.

Reply:
Yeah agreed. I'll remove the v7 bit from this.